docs/start/getting-started/fragments/ionic/data-model.md

```diff
@@ -3,11 +3,12 @@
 Add a [GraphQL API](https://docs.aws.amazon.com/appsync/latest/devguide/designing-a-graphql-api.html) to your app and automatically provision a database by running the the following command from the root of your application directory:
 
 ```bash
-amplify add api #accept defaults
+amplify add api
 ```
 
-The default values are highlighted below.
-```bash
+Accept the default values which are highlighted below:
```

Comment:
user studies have shown that people don't read much outside of codeblocks. we should make these either bold or retain them in comments